Source/controls/plrctrls.h

```diff
@@ -4,6 +4,7 @@
 #include <cstddef>
 #include <cstdint>
 
+#include "controls/devices/game_controller.h"

```

Comment:
this is an SDL2 only feature, apply guard

Reply:
Okay I get it. Ifndef.

Reply:
Still missing